go/violetear/go.mod

```
@@ -1,3 +1,3 @@
 module main
 
-require github.com/nbari/violetear v0.0.0-20190331194914-971f37295201
+require github.com/nbari/violetear df2078440951
```

you mean I can use 
```suggestion
require github.com/nbari/violetear v7.0.0
```
@nbari ?

I have
```
#10 0.545 go: errors parsing go.mod:
#10 0.545 /go/src/app/go.mod:3: require github.com/nbari/violetear: version "v7.0.0" invalid: should be v0 or v1, not v7
------
executor failed running [/bin/sh -c go get]: exit code: 1
```

---

Sorry, no idea how to fix it or make it work using version `7.0.0`

---

I've update the code then, but I'm pretty sure using a `v` prefix is something more standard in `go`